plugins/scaffolder-backend/sample-templates/pull-request/template.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: backstage.io/v1beta2
+kind: Template
+metadata:
+  name: pull-request
+  title: Pull Request Action template
+  description: scaffolder v1beta2 template demo publishing to PR on existing git repository
+spec:
+  owner: backstage/techdocs-core
+  type: service
+
+  parameters:
+    - title: Fill in some steps
+      required:
+        - name
+      properties:
+        name:
+          title: Name
+          type: string
+          description: Unique name of the component
+          ui:autofocus: true
+          ui:options:
+            rows: 5
+        description:
+          title: Description
+          type: string
+          description: Description of the component
+        targetPath:
+          title: Target Path in repo
+          type: string
+          description: Name of the directory to create in the repository
+    - title: Choose a location
```

Comment:
Wonder if we can make this use the `RepoUrlPicker` format instead?

Reply:
The reason it's like this is essentially that we don't 😅 Given our local assumptions that we are on github.com and everything is created in the same org, having the three fields visible is misleading (and people think 'owner' should be their team, not the Github org ... etc).

I'm happy to change the component to accept _either_ a repo URL or a separate host, org and name, and then update the template here to use the URL, if that would be helpful.

Reply:
Maybe actually it should be another component which is `RepoPicker`. That way we can grow that component over time to actually use your identity do so you can select repos that you have access to and it can do lookups wherever necessary.

Reply:
I have now changed the action so it accepts the separate properties or the URL.